client/src/locales/en-US.json

```diff
@@ -933,8 +939,8 @@
         "organization": "Organization",
         "workspaces": "My workspaces",
         "recentWorkspaces": "Recent",
-        "favorites": "Pinned",
-        "noFavorites": "No pinned workspaces",
+        "favorites": "Favorites",
```

Comment:
Didn't we change "Favorites" for "Pinned" like two months ago? Why changing it back? What does this have to do with this PR?